packages/contact-center/cc-components/src/components/task/CallControl/CallControlCustom/call-control-custom.utils.ts

```diff
@@ -57,6 +58,15 @@ export const createConsultButtons = (
         disabled: !consultCompleted,
         shouldShow: isAgentBeingConsulted && !!onTransfer,
       },
+      {
+        key: 'conference',
+        icon: 'call-merge-bold',
+        tooltip: 'Consult Conference',
+        onClick: handleConsultConferencePress || (() => {}),
+        className: 'call-control-button',
+        disabled: !consultCompleted,
+        shouldShow: isAgentBeingConsulted && !!handleConsultConferencePress,
```

Comment:
Nitpick: becasue its a boolean can we start the name with "is"
```suggestion
        isVisible: isAgentBeingConsulted && !!handleConsultConferencePress,
```

Reply:
addressed